examples/ffi/crashinfo.cpp

```diff
@@ -81,16 +98,18 @@ void add_stacktrace(ddog_crasht_Handle_CrashInfoBuilder *builder) {
                  "failed to add line");
 
     // This operation consumes the frame, so use .release here
-    check_result(ddog_crasht_StackTrace_push_frame(stacktrace.get(), new_frame.release(), true),
+    check_result(ddog_crasht_StackTrace_push_frame(stacktrace, new_frame.release(), true),
                  "failed to add stack frame");
   }
+  
+}
 
+void add_windows_style_frame(ddog_crasht_Handle_StackTrace* stacktrace) {
   // Windows style frame with normalization
   auto pbd_frame = extract_result(ddog_crasht_StackFrame_new(), "failed to make StackFrame");
-  check_result(ddog_crasht_StackFrame_with_ip(pbd_frame.get(), to_slice_c_char("0xDEADBEEF")),
+  check_result(ddog_crasht_StackFrame_with_ip(pbd_frame.get(), 3735928559), // 0xDEADBEEF
                "failed to add ip");
-  check_result(ddog_crasht_StackFrame_with_module_base_address(pbd_frame.get(),
-                                                               to_slice_c_char("0xABBAABBA")),
+  check_result(ddog_crasht_StackFrame_with_module_base_address(pbd_frame.get(), 2881137594), // 0xABBAABBA"
```

Comment:
`0x` constants should also work in C++, no? Or was the point to demonstrate that this is clearly an int not a string?

Reply:
omg, I forgot this one... good point, will fix it

Reply:
✔️